packages/graphql_server/package.json

```diff
@@ -14,11 +14,14 @@
     "serve": "$npm_package_config_env_dev && node dist/index.js"
   },
   "dependencies": {
+    "@types/node": "^18.16.3",
+    "axios": "^1.4.0",
+    "cheerio": "^1.0.0-rc.12",
     "fastify": "^4.17.0",
     "graphql": "^16.6.0",
     "mercurius": "^13.0.0",
-    "typescript": "^5.0.4",
-    "@types/node": "^18.16.3"
```

Comment:
do we not need @types/node?

Reply:
I am not exactly sure whether we need it or not

Reply:
Yes, it looks like it